Sizing clickstack updates (#6097)
ClickHouse/clickhouse-docs: docs/use-cases/observability/clickstack/managing/_snippets/_resource_estimation.md

When deploying ClickStack, provision compute to cover two independent workloads: **ingest** and **query**.

| Workload | Estimated resources |
|----------|---------------|
| **Ingest** | 1 vCPU per 10 MB/s of sustained ingest throughput |
| **Query** | 1 vCPU per 1 QPS and per 10 MB/s of sustained ingest throughput |

Assumptions:

- A **10x compression ratio** for storage - typically conservative for logs and traces.
- An average query response time of 2 seconds with a P99 of 5 seconds.

> - We assume most queries occur over recent data, with a mean lookback window of six hours, with a normal distribution and a standard deviation of one hour on either side. Users may wish to provision dedicated compute to query older data. In ClickHouse Cloud this can be idle (thus not incuring costs) when not in use.

Reviewer: ```suggestion
- We assume most queries occur over recent data, with a mean lookback window of six hours, with a normal distribution and a standard deviation of one hour on either side. You may wish to provision dedicated compute to query older data. In ClickHouse Cloud this can be idle (thus not incurring costs) when not in use.
```

> - We assume most queries occur over recent data, with a mean lookback window of six hours, with a normal distribution and a standard deviation of one hour on either side. Users may wish to provision dedicated compute to query older data. In ClickHouse Cloud this can be idle (thus not incuring costs) when not in use.

Reviewer: ```suggestion
- We assume most queries occur over recent data, with a mean lookback window of six hours, with a normal distribution and a standard deviation of one hour on either side. You may wish to provision dedicated compute to query older data. In ClickHouse Cloud this can be idle (thus not incuring costs) when not in use.
```

- While query compute can be scaled independently of ingest compute, it remains intrinsically linked to ingest volume. We assume as ingest increases, data density grows, resulting in larger scan volumes at query time and consequently higher query compute requirements.

:::note
These values are **estimates only** and should be used as an initial baseline. Actual requirements depend on query complexity, concurrency, retention policies, and variance in ingestion throughput. Always monitor resource usage and scale as needed.
:::

> The following table provides example sizings based on increasing ingest throughput in megabytes per second, alongside the corresponding data volumes in terabytes per month. This assumes a sustained average of **3 QPS** from ClickStack across all query types (search, dashboards, alerting). 

Reviewer: ```suggestion
The following table provides example sizings based on increasing ingest throughput in megabytes per second, alongside the corresponding data volumes in terabytes per month. This assumes a sustained average of **1 QPS** from ClickStack across all query types (search, dashboards, alerting). 
```

Should this not be 1 QPS?
